app/assets/javascripts/arclight/oembed_controller.js

```diff
@@ -18,7 +18,7 @@ export default class OembedController extends Controller {
         throw new Error(`HTTP error, status = ${response.status}`)
       })
       .then((body) => {
-        const oEmbedEndPoint = OembedController.findOEmbedEndPoint(body)
+        const oEmbedEndPoint = this.findOEmbedEndPoint(body)
```

Comment:
findOEmbedEndPoint is a static method. Doesn't in need to be invoked on a class or `this.constructor.findOEmbedEndPoint`?